Resources/Prototypes/Entities/Clothing/Uniforms/jumpskirts.yml

```
@@ -246,9 +246,11 @@
   description: The jumpskirt for the poor sop with a mop.
   components:
   - type: Sprite
-    sprite: Clothing/Uniforms/Jumpskirt/janitor.rsi
+#    sprite: Clothing/Uniforms/Jumpskirt/janitor.rsi # Commented out on Harmony
+    sprite: _Harmony/Clothing/Uniforms/Jumpskirt/janitor.rsi # Updated sprite pathing for Harmony
   - type: Clothing
-    sprite: Clothing/Uniforms/Jumpskirt/janitor.rsi
+#    sprite: Clothing/Uniforms/Jumpskirt/janitor.rsi # Commented out on Harmony
+    sprite: _Harmony/Clothing/Uniforms/Jumpskirt/janitor.rsi # Updated sprite pathing for Harmony
```

This can be simplified
```suggestion
    sprite: _Harmony/Clothing/Uniforms/Jumpskirt/janitor.rsi # Updated sprite pathing for Harmony
```